cmd/eskip/mediadefaults.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
I prefer the the version I implemented as it's really close to the StrategyPattern